source/slang/profile-defs.h

```diff
@@ -60,6 +60,14 @@ PROFILE_STAGE(Geometry, geometry,   SLANG_STAGE_GEOMETRY)
 PROFILE_STAGE(Fragment, fragment,   SLANG_STAGE_FRAGMENT)
 PROFILE_STAGE(Compute,	compute,    SLANG_STAGE_COMPUTE)
 
+PROFILE_STAGE(RayGeneration,    raygeneration,  SLANG_STAGE_RAY_GENERATION)
```

Comment:
Other stage names are single words. With these new stages we have multiple words. Perhaps using lower snake (so 'ray_generation') for the names be more consistent?